tools/regression_test.sh

```diff
@@ -222,7 +222,9 @@ function run_db_bench {
   db_bench_cmd="("'\$(which time)'" -p $DB_BENCH_DIR/db_bench \
       --benchmarks=$1 --db=$DB_PATH --wal_dir=$WAL_PATH \
       --use_existing_db=$USE_EXISTING_DB \
+      --perf_level=3 \
```

Comment:
Do we want to always enable perf_level 3, or should we also run with perf_level (default)?

Reply:
Make it an option? It can be set based on the regression test workload. Can be set to 3 for "myrocks" and default for "viewstate".